charts/capsule-proxy/values.yaml

```diff
@@ -2,17 +2,15 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-replicaCount: 1
-hostNetwork: false
-hostPort: false
-imagePullSecrets: []
+kind: Deployment # Deployment | DaemonSet
 image:
   repository: quay.io/clastix/capsule-proxy
   pullPolicy: IfNotPresent
   tag: ""
+imagePullSecrets: []
 options:
   listeningPort: 9001
-  logLevel: '5'
+  logLevel: '4'
```

Comment:
Why decrease from 5?

Reply:
Make it consistent with capsule log verbosity set to 4 as default